### PR TITLE
[NetAppFiles] update to 2023-11-01

### PR DESCRIPTION
**Pull request by audunn**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by audunn** [comment]
`dotnet build /t:GenerateSamples`  is failing but `dotnet build /t:GenerateCode` works  not sure why

File: sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,17 @@
 
 ### Other Changes
 
+## 1.6.0 ()
```
Comment (ArthurMa1978) [inline sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md:13]: When do you want to release the sdk?

Comment (audunn) [reply sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md:13]: asap since this includeds bugfixes.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**